web/src/integTest/java/com/microsoft/applicationinsights/framework/TestEnvironment.java

```diff
@@ -47,4 +45,33 @@ public String getApplicationStorageConnectionString() {
     public String getApplicationStorageExportQueueName() {
         return envProps.getProperty(KEY_APPLICATION_STORAGE_EXPORT_QUEUE_NAME);
     }
+
+    public static Properties loadPropertiesFromFile(String filePath) throws IOException
```

Comment:
This logic should reside in the PropertyUtils class that you removed so it can be reused for all property files used by the tests